portfolio.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+	<html lang="en-GB">
+	
+	<head>
+		<title>Charlotte Buckley - Portfolio</title>
+		<link rel="stylesheet" type ="text/css" href="style.css"> 
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+	</head>	
+	
+	<body>		
+		<nav class="nav-bar">
+			<a class="navigation-links" href="#home">home</a> 
+			<a class="navigation-links" href="#about-me">about me</a>
+			<button class="cv-button">download my CV!</button>
+			<a class="navigation-links" href="#portfolio">portfolio</a>
+			<a class="navigation-links" href="#contact-me">contact me</a>	
+		</nav>	
+	
+		<header id="home">
+			<div>
+				<span><img class="sparkle-inline" src="sparkles.gif" alt="sparkle">hi there</span> 
```

Comment:
need a closing / on your img tag (worth checking other imgs in this file too please)